doc/ext/sphinx_gallery/docs_resolv.py

```diff
@@ -0,0 +1,435 @@
+# -*- coding: utf-8 -*-
+# Author: Óscar Nájera
```

Comment:
+1 for the README. However, it will be removed after the next sphinx release, right?